### ISSUE TITLE
Recent projects shows only one, always the same project 

### ISSUE THREAD

**Issue by glocalglocal** [ccbogel/QualCoder#139]
Reported earlier but it may have been lost somewhere. 

Recent projects never showed more than one item. From what I see it's not the most recent one. At the moment it shows a specific one no matter which other projects I open and close.

**Comment by ccbogel** [comment]
This works fine on Ubuntu , will check on Windows 10.

**Comment by ccbogel** [comment]
As a thought - are you closing and opening qualcoder, or are you leaving it open?
as it will load recent projects list when opening qualcoder.
also, open and look at the recent project.txt file in the .qualcoder folder  - does this have multiple entries?

**Comment by glocalglocal** [comment]
After several days of experimenting, including closing and opening qualcoder, it started working a couple of days ago. Not sure what happened.

**Comment by glocalglocal** [comment]
There may be an issue. I opened/closed several projects. Without closing the application I checked Recent projects and there were only two listed. I checked recent_projects.txt in notepad and I got this error: 'Some number of solitary carriage returns were detected and removed.'. Is that a problem? 

On clicking OK, notepad showed correctly 4 projects. Recent Projects still showed 2 until I restarted qualcoder. I think Recent should show recent projects without us having to restart qualcoder.
